templates/jboss-eap.systemd.service.epp

```diff
@@ -0,0 +1,17 @@
+[Unit]
```

Comment:
Should we add a header like `# THIS FILE IS MANAGED BY PUPPET`?

Reply:
Done (in beiden systemd templates - wildfly und jboss)